internal/controller/certificates/policies/policies.go

```diff
@@ -74,6 +74,7 @@ func NewTriggerPolicyChain(c clock.Clock) Chain {
 		SecretIssuerAnnotationsNotUpToDate,
 		CurrentCertificateRequestNotValidForSpec,
 		CurrentCertificateNearingExpiry(c),
+		SecretKeystoreFormatMatchesSpec,
```

Comment:
@sathyanarays It may be as simple as moving this to 

https://github.com/cert-manager/cert-manager/blob/77c410f5cb99501e80dbdfcf8530a05610b165df/internal/controller/certificates/policies/policies.go#L92-L104

Which is instantiated here:

https://github.com/cert-manager/cert-manager/blob/77c410f5cb99501e80dbdfcf8530a05610b165df/pkg/controller/certificates/issuing/issuing_controller.go#L150-L153

Reply:
Done!